docs/ECLStandardLibraryReference/SLR-Mods/BLAS.xml

```diff
@@ -0,0 +1,1224 @@
+
+    <para></para>
+
+    <informaltable colsep="1" frame="all" rowsep="1">
+      <tgroup cols="2">
+        <colspec colwidth="80.50pt" />
+
+        <colspec />
+
+        <tbody>
+          <row>
+            <entry><emphasis>m</emphasis></entry>
+
+            <entry>Number of rows of matrix a</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>n</emphasis></entry>
+
+            <entry>Number of columns of matrix a</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>a</emphasis></entry>
+
+            <entry>Matrix a</entry>
+          </row>
+
+          <row>
+            <entry>Return:<emphasis></emphasis></entry>
+
+            <entry>Composite matrix of factors, lower triangle has an implied
+            diagonal of ones. Upper triangle has the diagonal of the
+            composite.</entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </informaltable>
+
+    <para>The <emphasis role="bold">dgetf2</emphasis> function produces a
+    combine lower and upper triangular factorization.</para>
+
+    <para>Example:</para>
+
+    <programlisting format="linespecific">IMPORT STD;
+STD.BLAS.Types.t_matrix test := [2,4,6,3,10,25, 9,34,100];
+STD.BLAS.dgetf2(3, 3, test); //result is [2,2,3,3,4,4,9,16,25];
+</programlisting>
+
+    <para></para>
+  </sect1>
+
+  <sect1 id="dpotf2">
+    <title>dpotf2</title>
+
+    <para><emphasis role="bold">STD.BLAS.dpotf2<indexterm>
+        <primary>STD.BLAS.dpotf2</primary>
+      </indexterm><indexterm>
+        <primary>BLAS.dpotf2</primary>
+      </indexterm><indexterm>
+        <primary>dpotf2</primary>
+      </indexterm>(</emphasis> <emphasis>tri,, r, A, clear);</emphasis>
+    <emphasis></emphasis></para>
+
+    <para></para>
+
+    <informaltable colsep="1" frame="all" rowsep="1">
+      <tgroup cols="2">
+        <colspec colwidth="80.50pt" />
+
+        <colspec />
+
+        <tbody>
+          <row>
+            <entry><emphasis>tri</emphasis></entry>
+
+            <entry>Indicates whether upper or lower triangle is used</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>r</emphasis></entry>
+
+            <entry>Number of rows/columns in the square matrix</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>A</emphasis></entry>
+
+            <entry>The square matrix A</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>clear</emphasis></entry>
+
+            <entry>Clears the unused triangle</entry>
+          </row>
+
+          <row>
+            <entry>Return:<emphasis></emphasis></entry>
+
+            <entry>The triangular matrix requested</entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </informaltable>
+
+    <para>The <emphasis role="bold">dgetf2</emphasis> function computes the
+    Cholesky factorization of a real symmetric positive definite matrix A. The
+    factorization has the form A = U**T * U , if UPLO = 'U', or * A = L *
```

Comment:
Looks like the comment block was bad. Change UPLO=A to tri set to Triangle.Upper and UPLO = L to Triangle.Lower